apps/condo/domains/common/components/Table/Filters.tsx

```diff
@@ -141,75 +193,71 @@ export const getGQLSelectFilterDropdown = (
                     style={containerStyles}
                 >
                     <GraphQlSearchInput
-                        style={GRAPHQL_SEARCH_INPUT_STYLE}
-                        search={search}
                         showArrow
-                        mode={mode}
-                        value={selectedKeys}
+                        {...gqlSelectProps}
+                        value={selectedKeys as any}
```

Comment:
why any?
<img width="575" alt="image" src="https://user-images.githubusercontent.com/1640424/223931326-3fbca718-a175-4eee-b095-6c33afb1247b.png">

Reply:
type Key = string | number

selectedKeys have type Key[]

type Key[] cannot cast to type string[]

Reply:
changed to @ts-ignore and added note about type casting problem